import_and_save.py

```diff
@@ -0,0 +1,85 @@
+import click
+import csv
+from importlib import import_module
+
+def save_into_mysql(filename, username, password):
+    try:
+        module = import_module('MySQLdb')
+        db_name = 'cli_test'
+        mydb_conn = module.connect(user=username, passwd=password)
+        cur = mydb_conn.cursor()
+        cur.execute("""CREATE DATABASE IF NOT EXISTS """ + db_name)
+        mydb_conn = module.connect(user=username, passwd=password, db=db_name)
+        cur = mydb_conn.cursor()
+        csv_data = csv.reader(file(filename))
+        lst = ['Name', 'Age', 'Address']
+        table_name = "People"
+        createsqltable = """CREATE TABLE IF NOT EXISTS """ + table_name + " (" + " VARCHAR(250),".join(lst) + " VARCHAR(250))"
+        cur.execute(createsqltable)
+        for row in csv_data:
+            cur.execute('INSERT INTO People(Name,Age,Address)''VALUES(%s, %s, %s)',row)
+        mydb_conn.commit()
+        cur.close()
+    except:
+        print('Sorry Something went wrong')
+
+def save_into_sqlite(filename):
+    try:
+        import sqlite3
+        con = sqlite3.connect("my_test.db")
+        cur = con.cursor()
+        cur.execute("CREATE TABLE IF NOT EXISTS t(Name, Age, Address);") # use your column names here
```

Comment:
duplicate name, age, address.. If I want to change the field age as dob, I have to change every other place.